stdlib/public/core/SetAlgebra.swift

```diff
@@ -189,7 +194,7 @@ public protocol SetAlgebra : Equatable, ExpressibleByArrayLiteral {
   ///   other means.
   @discardableResult
   mutating func insert(
-    _ newMember: Element
+    _ newMember: __consuming Element
   ) -> (inserted: Bool, memberAfterInsert: Element)
```

Comment:
Could this change to `mutating func insert(_ newMember: __consuming Element) -> Element?` (maybe `swap` would be a better name)? ... Although that's probably the same as `update`.